lib/import-export-visitor.js

```diff
@@ -652,48 +682,39 @@ function toModuleImport(source, specifierMap, uniqueKey) {
     const isLast = i === lastIndex;
     const locals = specifierMap[imported];
     const valueParam = safeParam("v", locals);
-    let bind = "";
 
     parts.push(
       JSON.stringify(imported),
-      ":function(", valueParam
+      ":(", valueParam
     );
 
     if (imported === "*") {
       // There can be only one namespace import/export specifier.
       assert.strictEqual(locals.length, 1);
-      let local = locals[0];
-
-      // We must be handling either an ImportNamespaceSpecifier or an
-      // ExportNamespaceSpecifier. We initialize the target object as the
-      // argument to a .bind method call on the setter function, so that
-      // we can refer to it as `this` inside the setter function.
-      bind = ".bind(" + local + "=Object.create(null))";
-      local = "this";
+      const local = locals[0];
 
+      if (local.includes(".")) {
```

Comment:
This might be easier to read if it was `local.startsWith("exports.")`, since that's what we care about.